includes/integration.php

```diff
@@ -145,7 +154,7 @@ public function check_for_keys()
                 return;
             }
             
-            $response = $this->activity_helper->create_activity('integrated', $this->activity_enpoint_domain);
+            $response = $this->activity_helper->create_activity('integrated', $this->activity_endpoint_domain);
```

Comment:
The `activity_endpoint_domain` doesn't seem to change. You can set it inside the activity helper class and stop passing it to the `create_activity` method.